app/components/safe_area_view/safe_area_view.ios.js

```diff
@@ -56,6 +53,8 @@ export default class SafeAreaIos extends PureComponent {
     }
 
     componentDidMount() {
+        this.navigationEventListener = Navigation.events().bindComponent(this);
```

Comment:
I know is not mandatory but shouldn't we apply it anyway to avoid potential memory leaks?

```
componentWillUnmount() {
    // Not mandatory
    if (this.navigationEventListener) {
      this.navigationEventListener.remove();
    }
  }
```